src/worker.h

```diff
@@ -34,6 +37,7 @@ class WorkerServiceImpl final : public WorkerService::Service {
   Status ExecuteTask(ServerContext* context, const ExecuteTaskRequest* request, ExecuteTaskReply* reply) override;
   Status ImportFunction(ServerContext* context, const ImportFunctionRequest* request, ImportFunctionReply* reply) override;
   Status Die(ServerContext* context, const DieRequest* request, DieReply* reply) override;
+  Status ImportReusableVariable(ServerContext* context, const ImportReusableVariableRequest* request, AckReply* reply) override;
```

Comment:
Does "import" make sense here given that you're not actually importing the variable? Off the top of my head I'd have used the word "define" instead.